libraries/data/include/mcrl2/data/detail/prover/smt_lib_solver.h

```diff
@@ -113,7 +114,7 @@ class SMT_LIB_Solver: public SMT_Solver
               v_sort_domain_list = sort_expression_list({v_sort});
               v_sort = sort_expression();
             }
-            for (auto v_sort_domain_elt : v_sort_domain_list)
+            for (sort_expression v_sort_domain_elt : v_sort_domain_list)
```

Comment:
Moet dit niet  for(const sort_expression& v_sort_domain_elt: v_sort_domain_list) zijn? Bij default verwacht ik const & als het type in een for loop. Deze vraag kan ook gesteld worden bij andere veranderingen in deze en vorige commits.

Reply:
Ik ben het hier wel mee eens, maar deze range based for loop was al zonder const reference en clang-tidy heeft die dus gewoon zo gelaten. Op sommige plekken waar clang tidy een iterator heeft omgezet naar een range based for loop zonder const reference komt dit doordat we dit soort constructies hebben:

```
for (application::const_iterator i=appl.begin(); i!=appl.end(); ++i)
{
  data_expression Argument= *i
```
clang-tidy is daarin dan conservatief met aanpassingen maken, en zet deze niet zelf om naar een const reference.

Reply:
Maar in deze gevallen kunnen we ze denk ik handmatig redelijk efficient omzetten.